projects/libsodium/secretbox_easy_fuzzer.cc

```diff
@@ -0,0 +1,29 @@
+extern "C" {
+  #include <sodium.h>
+}
+
+class SodiumState {
+public:
+  unsigned char key[crypto_secretbox_KEYBYTES];
+  unsigned char nonce[crypto_secretbox_NONCEBYTES];
+
+  SodiumState() {
+    sodium_init(); // this can fail with a non-zero return code
+    crypto_secretbox_keygen(key);
```

Comment:
See the comment I made on secret_key_auth_fuzzer.cc:12,
I think it would be better not to use `randombytes_buf` or `*_keygen` functions since they can cause indeterminism in the fuzzer. 
Can you use `randombytes_buf_deterministic` instead?